pod/playlist/static/js/playlist.js

```diff
@@ -141,16 +139,17 @@ document.addEventListener("DOMContentLoaded", function () {
               window.location.reload();
             } else {
               showalert(response.statusText, "alert-danger");
-              
             }
           }
         })
 
         .catch((error) => {
           showalert(
-            "Error deleting video. (" +
-              error +
-              ") The video could not be deleted.",
+            gettext(
+              "Error deleting video. (" +
+                error +
```

Comment:
attention : gettext ne fonctionne pas avec des chaines contenant des variables. dans ce cas, il est préférable de combiner gettext et interpolate.

Exemple pris ailleurs : 
var message = gettext("Caption for segment from %s to %s:");
    $("#captionTitle").text(
      interpolate(message, [
        FormatTime(theCaption.start),
        FormatTime(theCaption.end),
      ])
    );